src/main/java/com/hyperwallet/clientsdk/util/MultipartUtility.java

```diff
@@ -0,0 +1,119 @@
+package com.hyperwallet.clientsdk.util;
+
+import cc.protea.util.http.Response;
+import com.hyperwallet.clientsdk.HyperwalletException;
+
+import javax.xml.bind.DatatypeConverter;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.file.Files;
+
+public class MultipartUtility {
+    private HttpURLConnection httpConn;
+    private DataOutputStream request;
+    private final String boundary;
+    private final String crlf = "\r\n";
+    private final String twoHyphens = "--";
+
+    /**
+     * This constructor initializes a new HTTP POST request with content type
+     * is set to multipart/form-data
+     *
+     * @param requestURL
+     * @throws IOException
+     */
+    public MultipartUtility(String requestURL, String username, String password)
+            throws IOException {
+        final String pair = username + ":" + password;
+        final String base64 = DatatypeConverter.printBase64Binary(pair.getBytes());
+        // creates a unique boundary based on time stamp
+        boundary = this.twoHyphens+System.currentTimeMillis();
+        URL url = new URL(requestURL);
```

Comment:
Might be better to use a constant boundary.

Reply:
Resolved